addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11570,16 +11570,22 @@ msgstr ""
 #. label for "music/ballet/dance" epg subgenre value
```

Comment:
Please adapt the comment to match the change of this PR.

Reply:
@ksooo, are you sure? AFAICT, all other subgenres of the "music/ballet/dance" genre (5 of them) already have the identical comment:
```
#. label for "music/ballet/dance" epg subgenre value
```
Why should the missing subgenre get a different one?

Reply:
because the subgenre is Jazz

Reply:
Ah, now I see. You are right. Sorry fpr the noise.

Reply:
But look at the complete situation in strings.po. All subgenres of music/ballet/dance currently have exactly this comment. I understand this as "subgenre of music/ballet/dance". Are you asking me to change all of them to the actual term of the subgenre? And then, what about the other genres? Same situation there.